acceptance/Rakefile

```diff
@@ -44,29 +47,64 @@ task :acceptance do
   preserve_hosts = ENV["BEAKER_PRESERVEHOSTS"] || 'onfail'
   type = ENV["BEAKER_TYPE"] || 'foss'
   keyfile = ENV["BEAKER_KEYFILE"] || "#{ENV['HOME']}/.ssh/id_rsa-acceptance"
-  options_file = ENV["BEAKER_OPTIONSFILE"] || ''
-  test_suite = ENV["BEAKER_TESTSUITE"] || 'tests/'
-  load_path = ENV["BEAKER_LOADPATH"] || ''
-  pre_suite = ENV["BEAKER_PRESUITE"] || "setup/#{type}/pre_suite"
-  helper = ENV["BEAKER_HELPER"] || ''
-  post_suite = ENV["BEAKER_POSTSUITE"] || ''
+  test_suite = ENV["BEAKER_TESTSUITE"] || "#{type}/tests/"
+  pre_suite = ENV["BEAKER_PRESUITE"] || "#{type}/setup/pre_suite"
   opts = ENV["BEAKER_OPTS"] || ''
 
   beaker = "bundle exec beaker "
   beaker += " --xml"
   beaker += " --debug"
   beaker += " --no-color"
   beaker += " --repo-proxy"
-  beaker += " --preserve-hosts #{preserve_hosts}" if preserve_hosts != ''
   beaker += " --config #{config}" if config != ''
-  beaker += " --helper #{helper}" if helper != ''
+  beaker += " --preserve-hosts #{preserve_hosts}" if preserve_hosts != ''
   beaker += " --type #{type}" if type != ''
   beaker += " --keyfile #{keyfile}" if keyfile != ''
-  beaker += " --options-file #{options_file}" if options_file != ''
-  beaker += " --load-path #{load_path}" if load_path != ''
-  beaker += " --pre-suite #{pre_suite}" if pre_suite != ''
-  beaker += " --post-suite #{post_suite}" if post_suite != ''
   beaker += " --tests #{test_suite}" if test_suite != ''
+  beaker += " --pre-suite #{pre_suite}" if pre_suite != ''
   beaker += " #{opts}" if opts != ''
   sh beaker
 end
+
+namespace :ci do
+  namespace :test do
+
+    desc 'This task is used by CI to test the package pe-client-tools'
+    rototiller_task :package => [:set_test_target]do |t|
+
+      flags = [
+          {:name => '--xml', :is_boolean => true},
+          {:name => '--no-color', :is_boolean => true},
+          {:name => '--debug', :is_boolean => true},
+          {:name => '--hosts', :override_env => 'TEST_TARGET'},
+          {:name => '--preserve-hosts', :default => 'onfail', :override_env => 'BEAKER_PRESERVEHOSTS'},
+          {:name => '--keyfile', :default => "#{ENV['HOME']}/.ssh/id_rsa-acceptance", :override_env => 'BEAKER_KEYFILE'},
+          {:name => '--pre-suite', :default => 'pe/setup/pre_suite', :override_env => 'BEAKER_PRESUITE'},
+          {:name => '--tests', :default => 'pe/tests', :override_env => 'BEAKER_TESTSUITE'},
```

Comment:
This path is correct for defaulting to PE. However, if the `BEAKER_TYPE` is not set to `pe`, then the `scooter` gem is not installed. Oddly, however, the `beaker-pe` gem did install.

Ideally, we would not have to reference the `BEAKER_TYPE` variable in Jenkins since it does not exist for any of the other components in the matrix.

cc @zreichert